From 71eabbb5680f8b84d8846a53eb90dc0e15a2c308 Mon Sep 17 00:00:00 2001
From: Bjarni Ingi Gislason <bjarniig@rhi.hi.is>
Subject: [PATCH] curl/docs/libcurl/*: fix some formatting of man pages
To: 

  Fix some fomatting issues in man pages.

Details:

  From "mandoc -Tlint":

mandoc: curl_getdate.3:64:2: WARNING: skipping paragraph macro: PP empty
mandoc: curl_global_init_mem.3:56:2: ERROR: skipping end of block that is not open: RE
mandoc: curl_unescape.3:48:5: ERROR: skipping all arguments: br curl_easy_escape "(3)," curl_easy_unescape "(3)," curl_free "(3)," RFC 2396
mandoc: curl_unescape.3:48:2: WARNING: skipping paragraph macro: br after SH
mandoc: curl_url_cleanup.3:29:2: STYLE: fill mode already enabled, skipping: fi
mandoc: curl_url_dup.3:29:2: STYLE: fill mode already enabled, skipping: fi
mandoc: curl_url_set.3:32:2: STYLE: fill mode already enabled, skipping: fi

  From "test-groff -b -mandoc -T utf8 -rF0 -t -w w -z":

  [ "test-groff" is a developmental version of "groff" ]

troff: <curl_getdate.3>:108: warning: trailing space
troff: <curl_getdate.3>:109: warning: trailing space

Signed-off-by: Bjarni Ingi Gislason <bjarniig@rhi.hi.is>
---
 docs/libcurl/curl_getdate.3         | 5 ++---
 docs/libcurl/curl_global_init_mem.3 | 7 ++++---
 docs/libcurl/curl_unescape.3        | 2 +-
 docs/libcurl/curl_url_cleanup.3     | 6 +++---
 docs/libcurl/curl_url_dup.3         | 6 +++---
 docs/libcurl/curl_url_set.3         | 5 ++---
 6 files changed, 15 insertions(+), 16 deletions(-)

Index: curl/docs/libcurl/curl_getdate.3
===================================================================
--- curl.orig/docs/libcurl/curl_getdate.3
+++ curl/docs/libcurl/curl_getdate.3
@@ -63,7 +63,6 @@ first three letters. This is usually not
 If a decimal number of the form YYYYMMDD appears, then YYYY is read as the
 year, MM as the month number and DD as the day of the month, for the specified
 calendar date.
-.PP
 .SH EXAMPLE
 .nf
  time_t t;
@@ -111,5 +110,5 @@ On systems with 64 bit time_t: if the ye
 will return -1. (The Gregorian calendar was first introduced 1582 so no "real"
 dates in this way of doing dates existed before then.)
 .SH "SEE ALSO"
-.BR curl_easy_escape "(3), " curl_easy_unescape "(3), "
-.BR CURLOPT_TIMECONDITION "(3), " CURLOPT_TIMEVALUE "(3) "
+.BR curl_easy_escape "(3), " curl_easy_unescape "(3),"
+.BR CURLOPT_TIMECONDITION "(3), " CURLOPT_TIMEVALUE "(3)"
Index: curl/docs/libcurl/curl_global_init_mem.3
===================================================================
--- curl.orig/docs/libcurl/curl_global_init_mem.3
+++ curl/docs/libcurl/curl_global_init_mem.3
@@ -56,9 +56,10 @@ To replace realloc()
 To replace strdup()
 .IP "void *calloc_callback(size_t nmemb, size_t size);"
 To replace calloc()
-.RE
+.RS
 This function is otherwise the same as \fIcurl_global_init(3)\fP, please refer
 to that man page for documentation.
+.RE
 .SH CAUTION
 Manipulating these gives considerable powers to the application to severely
 screw things up for libcurl. Take care!
@@ -74,5 +75,5 @@ Added in 7.12.0
 CURLE_OK (0) means everything was ok, non-zero means an error occurred as
 \fI<curl/curl.h>\fP defines - see \fIlibcurl-errors(3)\fP.
 .SH "SEE ALSO"
-.BR curl_global_init "(3), "
-.BR curl_global_cleanup "(3), "
+.BR curl_global_init (3),
+.BR curl_global_cleanup (3)
Index: curl/docs/libcurl/curl_unescape.3
===================================================================
--- curl.orig/docs/libcurl/curl_unescape.3
+++ curl/docs/libcurl/curl_unescape.3
@@ -61,4 +61,4 @@ be removed in a future release.
 .SH RETURN VALUE
 A pointer to a null-terminated string or NULL if it failed.
 .SH "SEE ALSO"
-.br curl_easy_escape "(3)," curl_easy_unescape "(3)," curl_free "(3)," RFC 2396
+.BR curl_easy_escape "(3), " curl_easy_unescape "(3), " curl_free "(3), " RFC 2396
Index: curl/docs/libcurl/curl_url_cleanup.3
===================================================================
--- curl.orig/docs/libcurl/curl_url_cleanup.3
+++ curl/docs/libcurl/curl_url_cleanup.3
@@ -28,7 +28,7 @@ curl_url_cleanup - free a CURLU handle
 #include <curl/curl.h>
 
 void curl_url_cleanup(CURLU *handle);
-.fi
+.
 .SH DESCRIPTION
 Frees all the resources associated with the given CURLU handle!
 .SH EXAMPLE
Index: curl/docs/libcurl/curl_url_dup.3
===================================================================
--- curl.orig/docs/libcurl/curl_url_dup.3
+++ curl/docs/libcurl/curl_url_dup.3
@@ -28,7 +28,7 @@ curl_url_dup - duplicate a CURLU handle
 #include <curl/curl.h>
 
 CURLU *curl_url_dup(CURLU *inhandle);
-.fi
+.
 .SH DESCRIPTION
 Duplicates a given CURLU \fIinhandle\fP and all its contents and returns a
 pointer to a new CURLU handle. The new handle also needs to be freed with
Index: curl/docs/libcurl/curl_url_set.3
===================================================================
--- curl.orig/docs/libcurl/curl_url_set.3
+++ curl/docs/libcurl/curl_url_set.3
@@ -31,7 +31,6 @@ CURLUcode curl_url_set(CURLU *url,
                        CURLUPart part,
                        const char *content,
                        unsigned int flags)
-.fi
 .SH DESCRIPTION
 Given the \fIurl\fP handle of an already parsed URL, this function lets the
 user set/update individual pieces of it.
